testgrid/config/config.yaml

```diff
@@ -1990,159 +1990,66 @@ dashboards:
   - name: periodic-kubeadm-gce-1.6
     test_group_name: periodic-kubernetes-e2e-kubeadm-gce-1-6
 
-# Matches the list on https://github.com/kubernetes/test-infra/issues/2029
+# These are the release *blocking* tests.  These provide a valid binary signal
+# to gate releases (alpha, beta, official).
+# This list is used by kubernetes/release/lib/releaselib.sh to provide no/nogo
+# signal for releases.
+# The first test/suite in the list is the primary suite for release purposes
 - name: release-1.6-blocking
   dashboard_tab:
-  # Build/Verify
-  - name: build-1.6
-    test_group_name: ci-kubernetes-build-1.6
-  - name: verify-1.6
-    test_group_name: ci-kubernetes-verify-release-1.6
-  - name: test-go-1.6
-    test_group_name: ci-kubernetes-test-go-release-1.6
-  - name: federation-build-1.6
```

Comment:
Could you move this to release-1.6-all? It's not there yet and I don't want it to fall out entirely.